tests/test_failover.py

```diff
@@ -25,6 +27,11 @@ def _mock_db_conn(tmp_path, monkeypatch):
 def test_aws_region_failover(enable_all_clouds, _mock_db_conn, mock_aws_backend,
                              monkeypatch, capfd):
     """Test SkyPilot's ability to failover between AWS regions."""
+    # Set the debug info to True to print the debug info
+    monkeypatch.setattr(env_options.Options.SHOW_DEBUG_INFO, 'get',
```

Comment:
We have debug log enabled in CI by default before this PR https://github.com/skypilot-org/skypilot/actions/runs/14621593259/job/41022781855?pr=5329. I don't know why this PR will change this behavior, could you please help take a look? @zpoint